m3u8/model.py

```diff
@@ -181,6 +181,8 @@ def dump(self, filename):
             fileobj.write(self.dumps())
 
     def _create_sub_directories(self, filename):
+        if (os.path.isabs(filename) == False):
```

Comment:
Most style guides would prefer:

```suggestion
        if not os.path.isabs(filename):
```